lib/grape/validations/params_scope.rb

```diff
@@ -115,10 +115,15 @@ def required?
 
       # Adds a parameter declaration to our list of validations.
       # @param attrs [Array] (see Grape::DSL::Parameters#requires)
-      def push_declared_params(attrs)
+      def push_declared_params(attrs, param_alias = nil)
```

Comment:
Instead of `param_alias` I would make it `opts` and a `Hash` of different options, starting with `as`, for future proofing.